CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ All notable changes to this project will be documented in this file. From versio
 
 ### Fixed
 
+- Fix incorrect ordering of overloaded functions when schemas differ by @joelonsql in #4554
+  + The `Ord` instance for `Routine` had a typo comparing `schema2 == schema2` instead of `schema1 == schema2`.
```

Comment:
I think this should do? It's a small change and to explain it completely, we would otherwise need to write a _long_ description which might confuse the users.

```suggestion
- Fix incorrect ordering of overloaded functions by @joelonsql in #4555
  + Fixes comparison between two functions in the same schema differing by length of parameters.
```

Reply:
Sorry, I don't want to be pedantic, but as a user I still have no idea what that means for me. As a user, I don't compare functions - why would I? Also what is "length of parameters", do you mean number of parameters? Or the length of their names?

Also.. why are overloaded functions ordered in any way?

(I know the answers to some of these questions, I guess, but most of our users won't)

Reply:
(and to be clear, the reasons why I am not proposing a specific change is, because I just don't know what the actual difference is, so far. Comparing *schemas* wrong sure is a bit more scary - could it be that we'd be calling a function from a different schema in some cases? Or are we failing to call functions when they exist in the two different exposed schemas with the same name? Or.. I don't know.)